src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.txt

```diff
@@ -1,4 +1,6 @@
 Compat issues with assembly mscorlib:
+CannotChangeAttribute : Attribute 'System.Runtime.Versioning.UnsupportedOSPlatformAttribute' on 'System.IO.FileStream.Lock(System.Int64, System.Int64)' changed from '[UnsupportedOSPlatformAttribute("ios")]' in the contract to '[UnsupportedOSPlatformAttribute("freebsd")]' in the implementation.
```

Comment:
I just ran `dotnet build ./src/libraries/apicompat/ApiCompat.proj --no-incremental /p:UpdateBaselineFiles=true` to update this. Most of the unrelated changes appear to just be re-orderings.